### ISSUE TITLE
Launcher not starting on Steam Deck, shared library error.

### ISSUE THREAD

**Issue by chunkyjackson** [bfeber/HLA-NoVR-Launcher#19]
When I run the launcher in desktop mode nothing happens, when I load the launcher in the terminal it gives me this error. "error while loading shared libraries: /home/deck/HLAlyx/bin/../lib/libQt6QuickControls2.so.6: file too short

Other than that installing the mod manually does work.

**Comment by bfeber** [comment]
Yeah it's weird. It even works on the Steam Deck recovery image. Sadly I can't test it myself since I don't have a Steam Deck
